components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -22,18 +22,26 @@
     </dependencies>
 
     <extensions defaultExtensionNs="com.intellij">
-        <applicationService serviceImplementation="io.gitpod.jetbrains.remote.services.HeartbeatService" preload="true"/>
+        <applicationService serviceImplementation="io.gitpod.jetbrains.remote.services.HeartbeatService"
```

Comment:
I have enabled auto-formatting of the code on save which made these adjustments. There are no functional changes otherwise.

Happy to revert, but I prefer following the suggestions of IntelliJ :)